packages/gatsby-plugin-google-analytics/src/gatsby-node.js

```diff
@@ -54,4 +54,5 @@ exports.pluginOptionsSchema = ({ Joi }) =>
     queueTime: Joi.number(),
     forceSSL: Joi.boolean(),
     transport: Joi.string(),
+    disableWebVitalsTracking: Joi.boolean().default(true),
```

Comment:
We will make it a default in the next major. I want to do some more updates in a follow up PR

Reply:
NIT: maybe consider `webVitalsTracking` plugin option name (setting `false` would disable it) - booleans with negation in the name are quite confusing usually

Reply:
+1 on confusing naming

Reply:
👍 the goal is to enable it by default in the next major release after some TBT changes